homeassistant/components/abode/strings.json

```
@@ -10,12 +10,12 @@
       }
     },
     "error": {
-      "identifier_exists": "Account already registered.",
-      "invalid_credentials": "Invalid credentials.",
-      "connection_error": "Unable to connect to Abode."
+      "identifier_exists": "[%key:common::config_flow::abort::already_configured_account%]",
```

I would rename the key as well:
```suggestion
      "already_configured": "[%key:common::config_flow::abort::already_configured_account%]",
```

---

I checked in the code, this string is not used at all. Should I delete it?

---

Yes, you should.